src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -93,6 +102,26 @@
   &.pf-m-full-height {
     height: 100%;
   }
+
+  &.pf-m-danger {
+    --#{$empty-state}__icon--Color: var(--#{$empty-state}--m-danger__icon--Color);
+  }
+
+  &.pf-m-warning {
+    --#{$empty-state}__icon--Color: var(--#{$empty-state}--m-warning__icon--Color);
+  }
+
+  &.pf-m-success {
+    --#{$empty-state}__icon--Color: var(--#{$empty-state}--m-success__icon--Color);
+  }
+
+  &.pf-m-info {
+    --#{$empty-state}__icon--Color: var(--#{$empty-state}--m-info__icon--Color);
+  }
+
+  &.pf-m-custom {
+    --#{$empty-state}__icon--Color: var(--#{$empty-state}--m-custom__icon--Color);
+  }
 }
 
 .#{$empty-state}__content {
```

Comment:
```suggestion
.#{$empty-state}__content {
  row-gap: var(--#{$empty-state}__content--RowGap);
```

Reply:
@mattnolting Can you tell me more about what you expect here? __content isn't a flex container so RowGap wouldn't apply.

Reply:
I would expect our `content` containers to have a base layout, so they're flexible enough to include additional info. Any specific sub-components would then have padding to match the design spec. 

![Screenshot 2023-11-21 at 3 23 27 PM](https://github.com/patternfly/patternfly/assets/5385435/a0e93acc-a1ef-481c-a3ed-eadf0cdb1a1a)

In this example, the container provides base spacing of `spacer--md` for rows, where `pf-v5-c-empty-state__body` provides bottom padding.

Reply:
Also, I'd make `pf-v5-c-empty-state__body` `display: grid`